cmd/aws-s3-csi-mounter/csimounter/csimounter.go

```diff
@@ -62,12 +70,22 @@ func Run(options Options) (int, error) {
 	cmd := exec.Command(options.MountpointPath, args...)
 	cmd.ExtraFiles = []*os.File{fuseDev}
 	cmd.Env = options.MountOptions.Env
+
+	var stderrBuf bytes.Buffer
+
 	// Connect Mountpoint's stdout/stderr to this commands stdout/stderr,
 	// so Mountpoint logs can be viewable with `kubectl logs`.
 	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	cmd.Stderr = io.MultiWriter(os.Stderr, &stderrBuf)
 
-	klog.Info("Starting Mountpoint process")
+	exitCode, err := options.CmdRunner(cmd)
+	if err != nil {
+		// If Mountpoint fails, write it to `mountErrorPath` to let `PodMounter` running in the same node know.
+		if writeErr := os.WriteFile(mountErrorPath, stderrBuf.Bytes(), mountErrorFileperm); writeErr != nil {
+			klog.Infof("Failed to write error logs to %s: %v\n", mountErrorPath, err)
```

Comment:
Nit: we should mention these are mount-time error logs

Reply:
Addressed with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/347/commits/c32596f5165adde8f3b335c25cadeb02d0e25222